supersim.go

```diff
@@ -55,12 +59,27 @@ func (s *Supersim) Start(ctx context.Context) error {
 		return fmt.Errorf("l1 chain failed to start: %w", err)
 	}
 
+	var wg sync.WaitGroup
+	waitForAnvil := func(anvil *anvil.Anvil) {
+		defer wg.Done()
+		wg.Add(1)
+		utils.WaitForAnvilEndpointToBeReady(anvil.Endpoint(), 10*time.Second)
```

Comment:
we should avoid generic packages like `util` unless necessary.

How about adding a `Ready()` function to the `Anvil` struct. Since the implementation is domain specific anyways

Reply:
got it! just refactored the code so that both supersim & anvil have a `WaitUntilReady` function and got rid of the util package entirely